llvm/lib/Target/RISCV/RISCVVLOptimizer.cpp

```diff
@@ -257,16 +257,27 @@ static OperandInfo getOperandInfo(const MachineOperand &MO,
   // Vector Unit-Stride Instructions
   // Vector Strided Instructions
   /// Dest EEW encoded in the instruction and EMUL=(EEW/SEW)*LMUL
+  case RISCV::VLM_V:
```

Comment:
Please separate the mask instructions to their own review.  They're conceptual separate.